R/mod_hierarchical_count_table.R

```diff
@@ -608,6 +1104,10 @@ hierarchical_count_table_server <- function(
         row <- input[["cell_click"]][["row_id"]]
         col <- input[["cell_click"]][["column"]]
         subj_ids <- et()[["df"]][[col]][[row]][["subjid"]]
+
+        # If no subjects defined in the cell then do nothing
+        if (length(subj_ids) == 0) return()
```

Comment:
Not wrong, but I would argue that it would be more idiomatic to not use returns inside the observe, as it does not really return anything. I would use an if structure, `if(length(subjd_ids)>0) {code to show modal}`